docs_src/microservices/general/index.md

```diff
@@ -202,6 +205,40 @@ Thus, if your device name is 温度感测器 (a Simplified Chinese character str
 curl http://localhost:59881/api/{{ api_version }}/device/name/温度感测器
 ```
 
+### Enable the NameFieldEscape
+EdgeX services communicate with each other by HTTP API or MQTT broker, and the communicate might fail when using the HTTP or MQTT reserved characters. 
+
+- The URL reserved characters: `! * ' ( ) ; : @ & = + $ , / ? # [ ]`
+- The MQTT reserved characters: `# +`
+
+For example, the device name is `TestDevice/Virtual`, we should encode the `/` to `%2F` in curl command. 
+
+```shell
+curl http://localhost:59882/api/{{ api_version }}/device/name/TestDevice%2FVirtual/Bool
+```
+
+In this example, the command still failed because the device name contains the reserved character, and the Core Command didn't encode the `/` to `%2F` when sending the request to the Virtual device service, and the Virtual device service incorrectly retrieved the device name. 
+
+```shell
+{
+    "apiVersion": "{{ api_version }}",
+    "message": "request failed, status code: 404, err: {\"apiVersion\":\"v3\",\"message\":\"device TestDevice not found\",\"statusCode\":404}",
+    "statusCode": 404
+}
+```
+
+To resolve this issue, enable the `NameFieldEscape`.
```

Comment:
add the statement of limitation to use `%` character

Reply:
Added the note.